doc/book/admin/access_control.rst

```diff
@@ -71,35 +71,28 @@ system space with a
 `cryptographic hash function <https://en.wikipedia.org/wiki/Cryptographic_hash_function>`_
 so that, if the password is ‘x’, the stored hash-password is a long string
 like ‘lL3OvhkIPOKh+Vn9Avlkx69M/Ck=‘.
-When a client connects to a Tarantool instance, the instance sends a random
-`salt value <https://en.wikipedia.org/wiki/Salt_%28cryptography%29>`_
-which the client must mix with the hashed-password before sending
-to the instance. Thus the original value ‘x’ is never stored anywhere except
-in the user’s head, and the hashed value is never passed down a network wire
-except when mixed with a random salt.
+Tarantool supports two protocols for authenticating users:
 
-.. NOTE::
+*   `CHAP <https://en.wikipedia.org/wiki/Challenge-Handshake_Authentication_Protocol>`_ with ``SHA-1`` hashing
+
+    In this case, password hashes are stored in the ``_user`` space unsalted.
```

Comment:
The salting concept is never introduced on this page; shouldn't we add 1-2 sentences about it somewhere before?

Reply:
good point... I think adding a link to this article would be enough: https://en.wikipedia.org/wiki/Salt_(cryptography)